char command
---
float32 current_linear_vel

